edgeless_node/src/lib.rs

```diff
@@ -2,12 +2,14 @@
 // SPDX-FileCopyrightText: © 2023 Claudio Cicconetti <c.cicconetti@iit.cnr.it>
 // SPDX-FileCopyrightText: © 2023 Siemens AG
 // SPDX-License-Identifier: MIT
-use edgeless_api::orc::OrchestratorAPI;
+
+use edgeless_api::{node_registration::NodeCapabilities, orc::OrchestratorAPI};
```

Comment:
We need a policy for `use` statements now.
All Code I've written should follow a no `use` policy (with the exception of Traits) in accordance with Google's C++ Styleguide's ban on `using`, and the fact that I hate having to jump around while reading code.
Whenever you change stuff, you modify that to a random mix of use and explicit prefixes.

We need to agree on one of the two styles.

Reply:
We can do either, we should just agree on one one move towards that option.